pkg/apis/apps/v1alpha1/servicebindingrequest_types.go

```diff
@@ -52,10 +56,8 @@ type ServiceBindingRequestStatus struct {
 // BackingServiceSelector defines the selector based on resource name, version, and resource kind
 // +k8s:openapi-gen=true
 type BackingServiceSelector struct {
-	Group       string `json:"group"`
-	Version     string `json:"version"`
-	Kind        string `json:"kind"`
-	ResourceRef string `json:"resourceRef"`
+	schema.GroupVersionKind `json:",inline" yaml:",inline"`
```

Comment:
@isutton Could I assign https://issues.redhat.com/browse/APPSVC-324 to you 
CC @baijum ?

Reply:
Can we do this change in a separate PR merge it first?

Reply:
Sure if Igor is ok with it